app/controllers/members_controller.rb

```diff
@@ -11,7 +11,7 @@ def new
 
   def create
     @member = Member.new(member_params)
-
+    @member.email.downcase!
```

Comment:
This is something that should be implemented inside `Member`. Basically that class needs to become case-insensitive, and if there're any methods like `find_by_email` they should downcase the argument or do a case-insensitive search.